libs/core/src/lib/pagination/pagination.component.ts

```diff
@@ -137,6 +137,9 @@ export class PaginationComponent implements OnChanges, OnInit, OnDestroy {
     @Input()
     pageLabel = 'Page';
 
+    /** Aria label for the navigation element */
+    @Input()
+    ariaLabel = 'Pagination with id ' + this.id;
```

Comment:
Why do we use `Pagination with id {id}` as the default one?

Reply:
We need some label by default when multiple `<nav>`s are used and user does not specify `ariaLabel`. From the issue description and [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role#description),  it is not very clear whether the label has to be unique or not. To be on the safe side, I let it be unique by including the `id`.
![image](https://user-images.githubusercontent.com/53509521/124116618-c88ea400-da8c-11eb-88b6-f7eb46dbf7d2.png)

Reply:
just `Pagination`?

Reply:
just `Pagination` also works when I tried, but if the expectation is that each label must be unique, wouldn't having just `Pagination` reflect the same on all the different `<nav>`s?

Reply:
So if pagination must be unique then lib consumer must provide a unique descriptive `aria-label`, this is how I see it.
component.id is not descriptive by itself, it just generates something like `fd-pagination-0` `fd-pagination-1` and so on.
By default we would have `Pagination with id fd-pagination-0`, as for me it looks weird.

Reply:
I see, I guess it makes sense that we leave it to consumer to provide unique labels, if necessary. Have kept it as just `Pagination` now, please take a look.